localstack-core/localstack/cli/localstack.py

```diff
@@ -908,6 +909,10 @@ def print_profile() -> None:
             f" :bust_in_silhouette: [bold]Profile:[/bold] [blue]{', '.join(config.LOADED_PROFILES)}[/blue]"
         )
 
+def print_app() -> None:
+    console.print(
+        f" :globe_with_meridians: [bold]App:[/bold] \033https://app.localstack.cloud\033"
```

Comment:
**note**: Alternatively, we could use a short URL like https://localstack.dev/app so we can track usage.